docs/rules/no-tabs.md

```diff
@@ -1,10 +1,10 @@
-# Disallow tabs in file (no-tabs)
```

Comment:
If you are updating this title, this should also be updated in the rule's metadata. Otherwise rule index page is going to have different title.